tests/Mock/CancelSubscriptionSuccess.xml

```diff
@@ -130,6 +130,10 @@
         </nameValues>
         <subscriptionBalance xmlns="" xsi:type="xsd:decimal">0</subscriptionBalance>
         <statementFormat xmlns="" xsi:type="vin:StatementFormat">DoNotSend</statementFormat>
+        <cancelReason xmlns="" xsi:type="vin:CancelReason">
+          <reason_code xmlns="" xsi:type="xsd:string">[CANCEL_REASON]</reason_code>
```

Comment:
Should probably add this to the mock for CancelSubscriptionsSuccess as well

Reply:
good thought, but that response doesn't contain the autobills: https://github.com/vimeo/omnipay-vindicia/blob/master/tests/Mock/CancelSubscriptionsSuccess.xml

Reply:
ah yup it does not